test/test_variable_ops.cpp

```diff
@@ -119,11 +119,11 @@ TEST(VariableTest, SmallGraph2) {
 
   auto add = ops::Add(root.WithOpName("Add1"), var, c);
 
-  auto assign = ops::AssignAdd(root, var, add);
+  auto assign_add = ops::AssignAdd(root.WithOpName("AssignAdd"), var, add);
 
-  auto add2 = ops::Add(root.WithOpName("Add2"), assign, c);
+  auto add2 = ops::Add(root.WithOpName("Add2"), assign_add, c);
 
-  auto assign2 = ops::AssignSub(root, var, add2);
+  auto assign_sub = ops::AssignSub(root.WithOpName("AssignSub"), var, add2);
```

Comment:
Are we using the op names anywhere? Only some ops (Variable related) have names others don't.

Reply:
We are not, but I found it handy while debugging.